client_test.go

```diff
@@ -151,7 +208,7 @@ func TestClient_Read(t *testing.T) {
 	assert.NoError(t, c3.Initialize(ctx))
 	assert.NoError(t, c3.Read(ctx, fieldsToRead, cte1))
 	assert.Equal(t, cte1.ID, results["id"])
-	assert.Equal(t, cte1.Name, results["name"])
+	assert.NotEqual(t, cte1.Name, results["name"])
```

Comment:
Actually, it's due to the changed contract as in the commit comments. In this case, the "name" field wasn't requested, but the connector returned it anyway, which changed the value of the field. One could argue this was a test bug before, or at a minimum the contract between the connector and the client wasn't well defined.